CharacterCreation/02_Classes.txt

```diff
@@ -143,8 +143,11 @@ Skill Proficiencies:
 
 
 =====   DOCTOR   =====
-You know how or why something works [most likely humans / humanoid bodies), 
-and put that knowledge to good use by supporting your team.
+
+    You know the secrets of life and death. The very continued existance of 
+those around you depend on you striking the right balance between heal and harm. 
+One errant twitch of your hands could mean the difference between saving the day 
```

Comment:
"one errant twitch" is a vivid enough phrase that I'd only use it once in a short passage like this. m/b choose whether to use it here or in the Adrenaline Junkie flavor text.